src/types/Newsroom.ts

```diff
@@ -93,6 +93,7 @@ export interface Newsroom extends NewsroomRef {
         category: string | null;
     };
     ga_tracking_id: string | null;
+    segment_analytics_id: string | null;
```

Comment:
So this property was there all along? 🤔

Reply:
It wasn't exposed, like you mentioned. The column and PATCH API were implemented.